sample/subpkg/zcl_app.clas.abap

```diff
@@ -0,0 +1,15 @@
+class zcl_app definition
+  public
+  create public .
```

Comment:
I dont feel like these sample files is a good approach going forward, its not clear if they can be changed, if used for testing, what the fixture is if any.

instead, suggest to add more unit tests

Reply:
I can remove, if you insist.
The problem I was solving with it is a quick and short manual smoke test. All alternatives are worse (imho)
- npm link + running on some real project (e.g. abapgit, which is 100k lines long so not observable)
- `../abapmerge/abapmerge` from another folder + same issue with real projects
- if not a real project - then a folder with some similar sample code, but since it is not in repo, I must care about syncronizing it between machines

So in my opinion this sample is a handy thing. And it can be changed, no prob, it is just a playground right at hand. The changability may be described in readme. We can even convert it into a integration test (does mocha support snapshots ?) but then changability suffers so imho contraversial.

Reply:
not super important for this project

should make a real playground, but 🤷‍♂️